index.js

```diff
@@ -182,9 +182,21 @@ async function scrapeData() {
   return { locations };
 };
 
+function calculateActive(locations) {
+  return locations.map(l => {
+    let cases = l["cases"] !== undefined ? l["cases"] : 0;
+    let deaths = l["deaths"] !== undefined ? l["deaths"] : 0;
+    let recovered = l["recovered"] !== undefined ? l["recovered"] : 0;
+    l["active"] = cases - deaths - recovered;
+    return l;
+  });
+}
+
 async function writeData({ locations, featureCollection }) {
   let date = process.env['SCRAPE_DATE'] ?  '-' + process.env['SCRAPE_DATE'] : '';
 
+  locations = calculateActive(locations);
```

Comment:
Let's do this above inside of `scrape()`, that way it's available further down the pipeline and to the API.

Maybe you can add it to the loop that's currently calculating counties/states/countries so we're not looping twice.

Reply:
fixed